src/Microsoft.VisualStudio.ProjectSystem.Managed/Text/StringBuilderExtensions.cs

```diff
@@ -27,25 +27,23 @@ public static void AppendFormat(this StringBuilder builder, in StringFormat form
 
             int length = format.Length;
 
-            if (length == 0)
+            switch (length)
             {
-                builder.Append(format.Format);
-            }
-            else if (length == 1)
-            {
-                builder.AppendFormat(format.Format, format.Argument1);
-            }
-            else if (length == 2)
-            {
-                builder.AppendFormat(format.Format, format.Argument1, format.Argument2);
-            }
-            else if (length == 3)
-            {
-                builder.AppendFormat(format.Format, format.Argument1, format.Argument2, format.Argument3);
-            }
-            else
-            {
-                builder.AppendFormat(format.Format, format.Arguments);
+                case 0:
+                    builder.Append(format.Format);
+                    break;
+                case 1:
+                    builder.AppendFormat(format.Format, format.Argument1);
+                    break;
+                case 2:
+                    builder.AppendFormat(format.Format, format.Argument1, format.Argument2);
+                    break;
+                case 3:
+                    builder.AppendFormat(format.Format, format.Argument1, format.Argument2, format.Argument3);
+                    break;
+                default:
+                    builder.AppendFormat(format.Format, format.Arguments);
+                    break;
```

Comment:
nit: Technically, you could use a string expression here since they all return `StringBuilder`. I'm actually surprised this extension doesn't return `StringBuilder`, like how the rest of the methods of the same name do. I'd have to look at how this is used, but it might make sense to do switch expression, update the return type to return StringBuilder, and see if the callers have any issues.

Reply:
It looks like no one is using these extension methods. I'll delete them.